server/channels/manualtesting/manual_testing.go

```diff
@@ -111,12 +114,20 @@ func ManualTest(c *web.Context, w http.ResponseWriter, r *http.Request) {
 			} else {
 				c.Err = model.NewAppError("manualTest", "app.user.save.app_error", nil, "", http.StatusInternalServerError).Wrap(err)
 			}
+			return
+		}
 
+		if _, err := c.App.Srv().Store().User().VerifyEmail(user.Id, user.Email); err != nil {
```

Comment:
You need to reuse the `err` variable to avoid shadowing
```suggestion
		if _, err = c.App.Srv().Store().User().VerifyEmail(user.Id, user.Email); err != nil {
```